.gitignore

```diff
@@ -78,4 +78,7 @@ lib
 # tsc incremental build artifacts
 tsconfig.tsbuildinfo
 
-*.vsix
\ No newline at end of file
+*.vsix
+
+# Cached test resources
+test-packages/test-utils/.model-cache
```

Comment:
not really important or needs to be fixed...
I would consider a nested gitignore for such things in the future though.

Reply:
The problem is that eslint and prettier both use the top level gitignore file and don't support gitignore hierarchy.